src/components/CommentCard.jsx

```diff
@@ -24,6 +48,7 @@ function CommentCard( { commentsArray }) {
         )   
     }
 
+    if (isLoading) return <Loading />
```

Comment:
I'm not overly keen on this, pressing a delete button and seeing _all_ the comments disappear and be replaced with loading isn't ideal, I would perhaps render is loading lower down in the tree? On a per-comment basis would work really well. For example, you click delete and the deleting comment is replaced with a loading sign, then disappears.